.github/workflows/reusable-test.yml

```diff
@@ -0,0 +1,68 @@
+---
+name: Reusable test
+
+permissions: {}
+
+on:
+  workflow_call:
+    inputs:
+      python-version:
+        type: string
+        required: true
+      os:
+        type: string
+        required: true
+      continue-on-error:
```

Comment:
This is an implementation detail so I started calling it `xfail` to match pytest's semantics. Could be `allow-failures` or something like that.

Reply:
I chose `continue-on-error` because it matches [GitHub semantics](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error).